src/main/java/org/openmrs/uitestframework/test/TestBase.java

```diff
@@ -416,6 +417,12 @@ public EncounterInfo createTestEncounter(String encounterType, PatientInfo patie
         TestData.createEncounter(ei); // sets the uuid
         return ei;
     }
+    public static void createTestFileDocument() throws IOException{
+    	RandomAccessFile randomTestFileDocument = new RandomAccessFile("patientxxx.pdf","rw");
+    	randomTestFileDocument.writeChar('H');
+    	randomTestFileDocument.writeChars("This is a test file genersted randomly purposely for attachment during the clinival visit test");
+    	randomTestFileDocument.close();
+    }
```

Comment:
Worth looking into the code  indention to have something like ```js
{
public static void createTestFileDocument() throws IOException{
    	RandomAccessFile randomTestFileDocument = new RandomAccessFile("patientxxx.pdf","rw");
    	randomTestFileDocument.writeChar('H');
    	randomTestFileDocument.writeChars("This is a test file genersted randomly purposely for attachment during the clinival visit test");
    	randomTestFileDocument.close();  
  }
```

Reply:
Thanks I'll commit the changes here

Reply:
Can we move this utility method  to step class https://github.com/openmrs/openmrs-contrib-qaframework/blob/1545f1b411abb3d1843664ce00dec3228c4f8c0a/src/test/java/org/openmrs/contrib/qaframework/automation/Steps.java#L21 in qa framework since we are inheriting from steps class for all utility methods